flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -571,6 +585,10 @@ protected HighAvailabilityServices createHighAvailabilityServices(
         }
     }
 
+    public HighAvailabilityServices getHaServices() {
```

Comment:
Could this be protected, rather than public? It's only being used in a subclass.

Reply:
Good point, thanks